src/interfaces/wasm_bindgen/api.rs

```diff
@@ -36,8 +38,9 @@ pub async fn webassembly_search(
     master_key: Uint8Array,
     label_bytes: Uint8Array,
     keywords: ArrayOfKeywords,
-    max_results_per_keyword: usize,
-    max_depth: usize,
+    max_results_per_keyword: i32,
+    max_depth: i32,
```

Comment:
Why did you change these types?

Reply:
Because unsigned types doesn't have a really good representation in JS so I prefer the interface to match the real types of JS to not be surprise, for example, with the previous types, I didn't know what would happen if the JS sent a `-1` (since the TS type is just `number`)…